src/cls/IPM/Storage/Module.cls

```diff
@@ -561,6 +561,24 @@ Method LoadDependencies(pPhaseList As %List, ByRef pParams) As %Status
 		// Resolve and validate dependencies
 		Set tReloadSnapshots = +$Get(pParams("UpdateSnapshots"))
 		Set tPermitDowngrade = +$Get(pParams("PermitDowngrade"))
+
+		// When loading a module from a local folder, there might be a <mod root>/.modules/ folder containining dependencies.
+		// It's easier to configure a temporary repository than to handle this case in the dependency resolution code.
+		If $Get(pParams("RootDirectory")) '= "" {
```

Comment:
I like this idea. Some notes: 
- would be good to have this repo created as early as possible
- this repo should be the first in the list of repos to resolve against I believe? Or at least provide a way to signify its priority?
- should we clean up this repo? I think we probably should at the end of the load command. Since this could only happen from a load command, the creation and clean up of this repo could be part of load command in Main()?